### ISSUE TITLE
Az module not found in docker image azure-powershell:alpine-3.13

### ISSUE THREAD

**Issue by gpsarkar** [Azure/azure-powershell#16763]
### Description

Az modules have stopped working on image `mcr.microsoft.com/azure-powershell:alpine-3.13` . 
It was working earlier but suddenly stopped from 5/Jan/2022. 

I have changed the image to alpine-3.10 now, which is working.

### Issue script & Debug output

```PowerShell
This is an output of a powershell script running in that image alpine-3.13.

Name                           Value
----                           -----
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Linux 5.10.47-linuxkit #1 SMP Sat Jul 3 21:51:4…
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

Get-Package: No match was found for the specified search criteria and module names 'Az'.

Connect-AzAccount: /home/script.ps1:26
Line |
  26 |  Connect-AzAccount -Credential $cred -Tenant "$tenantID" -Subscription …
     |  ~~~~~~~~~~~~~~~~~
     | The term 'Connect-AzAccount' is not recognized as a name of a
     | cmdlet, function, script file, or executable program. Check
     | the spelling of the name, or if a path was included, verify
     | that the path is correct and try again.

New-AzApiManagementContext: /home/script.ps1:27
Line |
  27 |  $context = New-AzApiManagementContext -ResourceGroupName $azureResour …
     |             ~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The term 'New-AzApiManagementContext' is not recognized as a
     | name of a cmdlet, function, script file, or executable
     | program. Check the spelling of the name, or if a path was
     | included, verify that the path is correct and try again.
```

### Environment data

```PowerShell
PSVersion                      7.2.1
PSEdition                      Core
GitCommitId                    7.2.1
OS                             Linux 5.10.47-linuxkit #1 SMP Sat Jul 3 21:51:4…
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Module versions

```PowerShell
docker - azure-powershell:alpine-3.13
```

**Comment by dingmeng-xue** [comment]
Thanks for reporting. We found an expected error during building docker alpine image when we released Az 7.1.

We will fix it ASAP. For the time being, you can use tag `mcr.microsoft.com/azure-powershell:7.0.0-alpine-3.13` which contains Az 7.0. Tag `alpine-3.13` is still available but we stop updating it due to its version is old and not supported.

**Comment by dingmeng-xue** [comment]
Docker image are updated.  Please remove your original image and pull new version
```
docker run -it mcr.microsoft.com/azure-powershell:alpine-3.13
```
